db/schema.rb

```diff
@@ -91,6 +91,17 @@
     t.index ["user_id"], name: "index_journeys_on_user_id", using: :btree
   end
 
+  create_table "logs", force: :cascade do |t|
+    t.integer  "minutes"
```

Comment:
imparted_hours might be a better table_name? I see what you mean with class_logs but that sounds like a programming term, not like a business term

I'd use orders, it's a common term regardless of whether you 'ordered' something or not. I'd leave 'invoices' exclusively to refer to invoice documents (PDFs, numbering, etc)